src/App.tsx

```diff
@@ -1,9 +1,45 @@
 import './App.scss';
+import {
+  Routes, Route, Navigate, Link,
+} from 'react-router-dom';
+import React from 'react';
+import { PeoplePage } from './PeoplePage';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
+export const Home: React.FC = () => (
+  <div className="Home People">
+    <h2>Home page</h2>
   </div>
 );
 
-export default App;
+export const NotFoundPage: React.FC = () => (
+  <div className="NotFoundPage People">
+    <h2>Page not found</h2>
+  </div>
+);
+
+export const App = () => (
+  <>
+    <header className="Header">
+      <nav className="Nav">
+        <Link to="/home" className="Nav Nav__link">Home</Link>
+        &nbsp;
+        <Link to="/people" className="Nav Nav__link">People</Link>
+        &nbsp;
+      </nav>
+    </header>
+
+    <div className="App">
+      {/* <h1>People table</h1> */}
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route
+          path="/home"
+          element={<Navigate to="/" />}
```

Comment:
add replace attribute to not save /home route in [browser history](https://reactrouter.com/docs/en/v6/getting-started/concepts#history-and-locations) and avoid navigation loop

Reply:
done